azure-pipelines.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         displayName: nix cache key
       - task: CacheBeta@0
         inputs:
-          key: $(nix-cache-key)
+          key: $(nix-cache-key) | dQw4w9WgXcQ
```

Comment:
@stefanobaghino-da: do I want to know about this?

Reply:
[Urban diction entry on dQw4w9WgXcQ](https://www.urbandictionary.com/define.php?term=dQw4w9WgXcQ)

There was a problem with CI, this served as a test, will soon rebase on top of master when #3323 is merged (which it probably already is).

Reply:
Should be addressed by https://github.com/digital-asset/daml/pull/3320/commits/9e52f54a56b5569dc4f7f5898c1f6eb20f997d42

Will rebase on `master` if this is not enough.